### PR TITLE
Revert "expose IsPublic in CompleteAndCanonicalizeTlfName" and other tweaks

### PR DESCRIPTION
**Pull request by songgao**
Reverts keybase/client#4668

### CODE REVIEW

File: go/service/tlf.go [hunk 1]
```diff
@@ -60,21 +60,14 @@ func (h *tlfHandler) CompleteAndCanonicalizeTlfName(ctx context.Context, arg key
 	// Append username in case it's not present. We don't need to check if it
 	// exists already since CryptKeys calls below transforms the TLF name into a
 	// canonical one.
-	arg.Query.TlfName = arg.Query.TlfName + "," + string(username)
+	arg.TlfName = string(username) + "," + arg.TlfName
```
Comment (patrickxb) [inline go/service/tlf.go]: Change comment to Prepend.

Can you also mention that this makes username a writer on this TLF, which might be unexpected?

Comment (patrickxb) [inline go/service/tlf.go]: Do we need a follow-up ticket to investigate how to handle reader TLFs better?

Comment (songgao) [inline go/service/tlf.go]: Sounds good! I'll make the ticket

**Pull request merged**